### PR TITLE
Add a button to export the referent's user list to a csv file

### PR DESCRIPTION
**Pull request by rpg600**

### CODE REVIEW

File: app/Resources/views/referent/users/_list.html.twig [hunk 1]
```diff
@@ -93,8 +93,13 @@
                 <span id="selected-users-count">0</span> utilisateur(s) sélectionnés
             </span>
 
+            <div id="csv-export">
+                <a class="btn btn--blue referent__btn-export-csv disabled" id="csv-export-btn">
+                    Export csv
```
Comment (hhamon) [inline app/Resources/views/referent/users/_list.html.twig]: Better to write "CSV" in capital case?

File: front/pages/referent_users.js [hunk 2]
```diff
@@ -11,20 +12,48 @@ export default (slugifier, columns, users) => {
...
 
+        render(
+            <CSVLink 
+                data={[headerNames, ...data]} 
+                filename={filename} 
+                className={"btn btn--blue referent__btn-export-csv" + (0 === count ? ' disabled' : '')} 
+                id="csv-export-btn" 
+                onClick={(0 === count) ? (e) => e.preventDefault() : ''}
+            >
+                Export csv
```
Comment (hhamon) [inline front/pages/referent_users.js]: Better to write "CSV" in capital case?

File: app/Resources/views/referent/users/_list.html.twig (hunk 1 above)
Comment (rpg600) [inline app/Resources/views/referent/users/_list.html.twig]: An uppercase style is applied here.

Comment (HeahDude) [inline app/Resources/views/referent/users/_list.html.twig]: Still the "E" respects the case, so should CSV.

**Comment by hhamon** [comment]
Shall we merge?

**Pull request merged**